horace_core/sqw/coord_transform/@aProjectionBase/aProjectionBase.m

```diff
@@ -92,8 +92,9 @@
         alatt_defined
         % returns true if lattice angles have been set up
         angdeg_defined
-        % old interface to img_offset for old scripts accepting a
-        % structure, containign this value
+        % old interface to img_offset for old data containing a
+        % structure with this value or old user scripts, which define
```

Comment:
```suggestion
        % structure with this value, or old user scripts which define
```
I've fixed the punctuation, but "this value" still hasn't been defined. Presumably old data had a structure with uoffset in it and now you have a property to get/set it but you need to define what uoffset is and how it relates to img_offset

Reply:
the property value?

Reply:
we are talking about the property and its value